pkg/queue/cluster_queue_impl_test.go

```diff
@@ -102,20 +102,6 @@ func Test_Delete(t *testing.T) {
 	}
 }
 
-func Test_Dump(t *testing.T) {
```

Comment:
But this test itself is not very comprehensive. Since we use Dump in a lot of the unit tests here, I think it's just repetitive to have a test exclusive for dump.